noxfile.py

```diff
@@ -52,3 +52,14 @@ def build(session, autobuild=False):
 def preview(session):
     session.install("sphinx-autobuild")
     build(session, autobuild=True)
+
+
+@nox.session(py="3")
+def linkcheck(session):
+    session.install("-r", "requirements.txt")
+    session.run(
+        "sphinx-build", 
+        "-b", "linkcheck", # use linkcheck builder
```

Comment:
Use the same options as other sessions
```suggestion
        "-b", "linkcheck", # use linkcheck builder
        "--color",
        "-n", "-W", "--keep-going",  # be strict
```